src/main/java/org/broadinstitute/hellbender/tools/gvs/extract/ExtractCohortEngine.java

```diff
@@ -208,27 +209,27 @@ public void traverse() {
         }
         final String rowRestrictionWithFilterSetName = rowRestriction + " AND " + SchemaUtils.FILTER_SET_NAME + " = '" + filterSetName + "'";
 
-        boolean noVqslodFilteringRequested = VQSLODFilteringType.equals(ExtractCohort.VQSLODFilteringType.NONE);
-        if (!noVqslodFilteringRequested) {
-            // ensure vqslod filters are defined. this really shouldn't ever happen, said the engineer.
-            if (vqsLodSNPThreshold == null || vqsLodINDELThreshold == null) {
-                throw new UserException("Vqslod filtering thresholds for SNPs and INDELs must be defined.");
+        boolean noVQScoreFilteringRequested = vqScoreFilteringType.equals(ExtractCohort.VQScoreFilteringType.NONE);
+        if (!noVQScoreFilteringRequested) {
+            // ensure vqScore filters (vqsLod or Sensitivity) are defined. this really shouldn't ever happen, said the engineer.
```

Comment:
dont we have defaults?

Reply:
Yeah, we do. I didn't write that comment, but I can dig a little deeper to see if it could.